shift_detector/checks/embedding_distance_check.py

```diff
@@ -22,10 +22,10 @@ def run(self, store):
                                        "\tDistance between Datasets: {}"\
                 .format(data[column_name][0], data[column_name][1], data[column_name][2])
 
-            if (abs(data[column_name][0] - data[column_name][1]) > data[column_name][0] * 3
-                    or abs(data[column_name][0] - data[column_name][1]) > data[column_name][1] * 3
-                    or abs(data[column_name][0] - data[column_name][2]) > data[column_name][0] * 3
-                    or abs(data[column_name][1] - data[column_name][2]) > data[column_name][1] * 3):
```

Comment:
Even it's not directly written in the PEP 8 documentation, the line break before the operator seems to be preferred. https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator

But since the last paragraph states

> In Python code, it is permissible to break before or after a binary operator, as long as the convention is consistent locally. For new code Knuth's style is suggested.

It's ok....